config/config.go

```diff
@@ -138,8 +138,12 @@ type ServerConfig struct {
 	LogDockerTail string `ini:"db_log_docker_tail"`
 
 	// Configures the collector to start a built-in syslog server that listens
-	// on the specifed "hostname:port" for Postgres log messages
+	// on the specified "hostname:port" for Postgres log messages
 	LogSyslogServer string `ini:"db_log_syslog_server"`
+	// For TLS support for syslog drain
+	LogSyslogServerCAPath   string `ini:"db_log_syslog_server_ca_path"`
```

Comment:
No strong feelings on the first point, but for context, we're using the `db_ssl*` setting names for Postgres to match Postgres' own parameters. If syslog has conventions here, it would be good to match them.

I think it would be good to allow specifying the contents, if only to match our other settings.

Reply:
Good to know the context. I think syslog doesn't really have the thing, but following opentelemetry's naming sounds like a good pattern, so I'll match to https://pkg.go.dev/github.com/open-telemetry/opentelemetry-collector-contrib/receiver/syslogreceiver#section-readme

Reply:
(also I'll skip providing contents for now, it's easy to add if we receive the feature request of it)

Reply:
The naming makes sense, though I do think we should provide a way to specify cert contents: it's awkward to make this work in a container without that.

Reply:
Got it, I'll add `_contents` thingy 👍

Reply:
Thanks for adding these!